src/error/CustomValidationError.ts

```diff
@@ -0,0 +1,12 @@
+export class CustomValidationError extends Error {
```

Comment:
Should we find a better name now that it's in akashjs? Something more descriptive like `AkashValidationError` or `SDLValidationError` maybe?

Reply:
I think `SDLValidationError` fits better

Reply:
Totally agree, will change